distro/configs/bahmniemr/eip/routes/write-bahmniOrder-as-testOrder.xml

```diff
@@ -0,0 +1,35 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+    <route id="write-bahmniOrder-as-testOrder" errorHandlerRef="watcherErrorHandler">
+        <from uri="direct:write-bahmniOrder-as-testOrder"/>
+        <setProperty name="bahmniTestOrderTypeUuid">
+            <simple>${properties:bahmni.test.orderType.uuid:'xxxxx'}</simple>
```

Comment:
Let's provide a valid default/placeholder or default to empty

Reply:
Well, this is a little more explicitly clear that there's no UUID config provided. Having it empty might be ambiguous to someone looking at the code. But I guess alright to leave it empty.

Reply:
Also I was trying to avoid the complications that might arise with comparing an empty string(s).

Reply:
Maybe, I could replace the string with a more descriptive one: `uuid-not-existing`.